lms/djangoapps/support/static/support/js/collections/certificate.js

```diff
@@ -0,0 +1,22 @@
+;(function (define) {
+    'use strict';
+    define(['backbone', 'support/js/models/certificate'],
+        function(Backbone, CertModel) {
+            var CertCollection = Backbone.Collection.extend({
```

Comment:
Instead of creating a variable and returning it that I would just return Backbone.Collection.extend({....